scss/bitstyles/base/font-face/_.scss

```diff
@@ -25,15 +25,15 @@
       $bitstyles-font-family-webfont,
       normal,
       $bitstyles-font-weight-medium,
-      $bitstyles-webfont-font-url
+      #{$bitstyles-webfont-font-url + '-' + $bitstyles-font-weight-medium}
     );
   }
   @if (list.index($bitstyles-webfont-variants, 'bold')) {
     @include webfont(
       $bitstyles-font-family-webfont,
       normal,
       $bitstyles-font-weight-bold,
-      $bitstyles-webfont-font-url
+      #{$bitstyles-webfont-font-url + '-' + $bitstyles-font-weight-bold}
```

Comment:
Wouldn't it make more sense if the url would all be variables and you would construct them in `typography`

from a `$bitstyles-webfont-font-url-base`? to set the defaults? Like this one could override this with fonts that do not match the naming patterns if one wishes :)

Reply:
Yeh this part was bugging me too. Thanks for the poke, I’ve updated it — I’m now specifying a set of fonts with their properties as a sass map, and this file iterates over it. If the map is empty, no `@fontface`s are output